serpent-moltres/full-core/extract-convert.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python3
+# This file is based in $MOLTRES/python/extractSerpent2GCs.py
```

Comment:
I would prefer that modifications should be made in that file to improve it (not here, where it is just a violation of DRY that will be extremely frustrating to untangle.)